sdk/src/main/java/com/bugsnag/android/FileStore.java

```diff
@@ -69,7 +72,8 @@ String write(@NonNull T streamable) {
 
         Writer out = null;
         try {
-            out = new FileWriter(filename);
+            FileOutputStream fos = new FileOutputStream(filename);
+            out = new BufferedWriter(new OutputStreamWriter(fos, "UTF-8"));
 
             JsonStream stream = new JsonStream(out);
             stream.value(streamable);
```

Comment:
not related to this PR but should we be closing this stream on the next line in the `finally` block?

Reply:
Yes, that would make more sense than closing two separate streams. I've updated the PR so that we do this.